### PR TITLE
Logo's

### PR DESCRIPTION
**Pull request by JakeDaBoss**
This adds ~~two~~ one new logo ~~'s~~, and deletes the base NT one.

Instructions in Changelog.

### CODE REVIEW

File: code/_helpers/text.dm [hunk 1]
```diff
@@ -421,7 +421,8 @@ proc/TextPreview(var/string,var/len=40)
 	t = replacetext(t, "\[/grid\]", "</td></tr></table>")
```
Comment (NanakoAC) [inline code/_helpers/text.dm:421]: Why are these files all showing up \] as changed?

File: code/modules/codex/entries/paperwork.dm [hunk 2]
```diff
@@ -22,7 +22,8 @@
 \[hr\] : Adds a horizontal rule.<br>
 \[list\] - \[/list\] : A list. (Does not work with crayons)<br>
 \[*\] : A dot used for lists. (Does not work with crayons)<br>
-\[logo\] - Inserts EXO logo image.<br>
+\[logo\] - Inserts CEC logo image.<br>
```
Comment (NanakoAC) [inline code/modules/codex/entries/paperwork.dm]: these macros are a bit confusing, they're both the cec logo. maybe change the second one to be more descriptive of context

Which one is normal and which is formal

File: code/_helpers/text.dm (hunk 1 above)
Comment (JakeDaBoss) [reply code/_helpers/text.dm:421]: Unknown, but they're all complete. http://prntscr.com/tliac6

File: code/modules/codex/entries/paperwork.dm (hunk 2 above)
Comment (JakeDaBoss) [reply code/modules/codex/entries/paperwork.dm]: Fixed in #be1b184

**Comment by JakeDaBoss** [comment]
Requested fixes provided.

**Comment by NanakoAC** [comment]
do these fit the same size as the NT logos? they look rather large

**Comment by JakeDaBoss** [comment]
> do these fit the same size as the NT logos? they look rather large



> do these fit the same size as the NT logos? they look rather large

They do now.

Alternative logo, the "formal" one as intended was removed for now. I'll re-add this in a different PR, in due time.

For now, the [logo] format will place the grey C with red dot.

File: code/modules/codex/entries/paperwork.dm [hunk 3]
```diff
@@ -22,7 +22,8 @@
 \[hr\] : Adds a horizontal rule.<br>
 \[list\] - \[/list\] : A list. (Does not work with crayons)<br>
 \[*\] : A dot used for lists. (Does not work with crayons)<br>
-\[logo\] - Inserts EXO logo image.<br>
+\[logo\] - Inserts regular CEC logo image.<br>
+\[ceclogo\] - Inserts the formal CEC Logo image. This logo is only used for official, outbound or external documents. Not commonly used internally.<br>
```
Comment (NanakoAC) [inline code/modules/codex/entries/paperwork.dm]: help text for formal logo is here

fixing it would be best, but if not, remove this text for now

**Review by NanakoAC — CHANGES_REQUESTED** [review]
helptext

File: code/modules/codex/entries/paperwork.dm (hunk 3 above)
Comment (JakeDaBoss) [reply code/modules/codex/entries/paperwork.dm]: Fixed.

**Pull request merged**